pkg/cmd/roachtest/github.go

```diff
@@ -292,6 +313,12 @@ func (g *githubIssues) createPostRequest(
 				"non-metamorphic run, there should be a similar issue without the "+metamorphicLabel+" label. If there "+
 				"isn't one, it is possible that this failure is caused by a metamorphic constant.")
 	}
+	if runtimeAssertionsBuild {
+		topLevelNotes = append(topLevelNotes,
+			"This build has runtime assertions enabled. If the same failure was hit in a "+
+				"non-cockroachEA run, there should be a similar issue without the "+runtimeAssertionsLabel+" label. If there "+
```

Comment:
Nit: `non-cockroachEA` might not be an obvious term for some. Maybe "was hit in a run without assertions enabled, ..."